pkg/asset/manifests/vsphere/infrastructure.go

```diff
@@ -103,15 +106,33 @@ func GetInfraPlatformSpec(ic *installconfig.InstallConfig, clusterID string) *co
 	return &platformSpec
 }
 
-// vipsToCIDRs takes a single ip address and converts it to CIDR notation.
-func vipsToCIDRs(vips []string) []configv1.CIDR {
-	cidrs := make([]configv1.CIDR, len(vips))
-	for i, vip := range vips {
-		mask := "/32"
-		if utilsnet.IsIPv6String(vip) {
-			mask = "/128"
+// vipsToCIDR takes a single ip address and converts it to CIDR notation.
```

Comment:
```suggestion
// vipsToCIDRs takes VIP addresses and converts them to CIDR notation.
```

nit: This is a suggestion if you'd like to go with the direction commented here: https://github.com/openshift/installer/pull/9743#discussion_r2107790766